spec/commands/inject_spec.rb

```diff
@@ -52,6 +52,24 @@
     end
   end
 
+  context "with source option" do
+    it "add gem with source option in gemfile" do
+      bundle "inject 'foo' '>0' --source file://#{gem_repo1}"
+      gemfile = bundled_app("Gemfile").read
+      str = "gem 'foo', '> 0', :source => 'file://#{gem_repo1}'"
+      expect(gemfile).to include str
+    end
+  end
+
+  context "with group option" do
+    it "add gem with group option in gemfile" do
```

Comment:
Thanks for review I have added the testcase for multiple groups in [this commit ](https://github.com/bundler/bundler/pull/5456/commits/cfe4eccfc33a8f6372a7197ffae95c15e023df95)